### DISCUSSION TITLE
Problem with libraries in OpenModelica v1.22

### DISCUSSION THREAD

**Discussion by ejperezHI** [OpenModelica/OpenModelica#11561] [Q&A]
Hello everyone,

I have recently installed succesfully (apparently) OpenModelica 1.22 in my Ubuntu 20.04 LTS computer. However, when I navigate through the tree on the left, instead of appearing the "logos" of each type of elements (electrical, fluids, thermal...etc), every single element of the tree has a "P" on it, also a "translation error" appears on the message browser.
Also, I cannot create any model (because of that, I guess).

![image](https://github.com/OpenModelica/OpenModelica/assets/131266350/b2e7dc21-ad4c-47ab-b160-e42d5dfe7816)
![image](https://github.com/OpenModelica/OpenModelica/assets/131266350/678e84ba-0e55-4d8e-9b8e-11d08df4c480)

How can I solve this? I have tried using the (I think that in the right way) "install library" option, "installing" the "complex library", the "modelica library" etc. But it says me that those libraries are already installed.

How can I solve this problem and start using and OpenModelica as usual?

Thanks in davance,

**Comment by casella** [comment]
According to the discussion in #11548, installing omc-common should help. If so, please close this, otherwise please give feedback.
